tests/test_examples.py

```diff
@@ -317,6 +317,8 @@ def _run_installer(
             check=check_subprocess,
             options=options,
         )
+        if request and ON_CI:
+            request.addfinalizer(lambda: shutil.rmtree(str(install_dir), ignore_errors=True))
```

Comment:
Should we do this for all installers? Regardless, let's add a comment that we are doing this to free space and prevent "running out of storage" errors.